tutorials/5-docker-intro/index.md

```diff
@@ -53,33 +61,35 @@ For the avoidance of doubt:
 
 What if you do not have a Linux operating system? Not to worry, Docker simplifies your life by installing and running a virtual machine running a barebones Linux on your host computer.
 
-After installation, when you want to use Docker, you _start Docker_, i.e. you start the virtual machine running Linux, which is the part that takes time. After it has started you can use commands to run containers; and when you no longer need to use Docker, you can stop it and regain the memory it used.
+After installation, when you want to use Docker, you _start Docker_ - you start the virtual machine running Linux, which is the part that takes time. After it has started, you can use commands to run containers. When you no longer need to use Docker, you can stop it and regain the memory it used.
+
+In these tutorials, you will come across a lot of Docker commands, so it makes sense to familiarize yourself with them.
 
-In these tutorials, you will come across a lot of Docker commands, so it makes sense to familiarize yourself with it. First, [install it](https://www.docker.com/get-started).
+First, [install it](https://www.docker.com/get-started). Next start Docker.
 
-Next start Docker. When it has started you can run your first container. For instance, with Node.js' `lts-slim` image:
+When it has started you can run your first container. For instance, with Node.js' `lts-slim` image:
 
 ```sh
 $ docker run -it node:lts-slim
 ```
 
-`-it` is short for `--interactive --tty` and means "with input and output" instead of a fully detached container. Learn more with `docker run --help`. This should return you a Node prompt:
+`-it` is short for `--interactive --tty` and means "with input and output", instead of a fully detached container. Learn more with `docker run --help`. This should return you a Node (Node.js) prompt:
```

Comment:
```suggestion
`-it` is short for `--interactive --tty` and means "with input and output", instead of a fully detached container. Learn more with `docker run --help`. This should return you a Node.js prompt:
```